doc_admin/AdminManual.adoc

```diff
@@ -381,10 +388,14 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways. The easiest way is to use the provided scripts (oh.sh/oh.ps1) by choosing the option "-s":
+
+Restore of the database is also possible; See script help form more informations.
```

Comment:
`See script help form more informations.` should be `see script help for more information.`

Reply:
fixed